config/core/configmaps/autoscaler.yaml

```diff
@@ -80,7 +80,7 @@ data:
     # -1 denotes unlimited target-burst-capacity and activator will always
     # be in the request path.
     # Other negative values are invalid.
-    target-burst-capacity: "200"
+    target-burst-capacity: "0"
```

Comment:
This won't help though, will it? You gotta reset it in the reading part of this.

Reply:
well it it'll help for tests that don't override it.

Reply:
It doesn't I think? This is the example section of the config 🤔